includes/class-sensei-content-drip.php

```diff
@@ -112,6 +112,10 @@ public function __construct() {
 		$this->assets_url    = esc_url( trailingslashit( plugins_url( '/assets/', SENSEI_CONTENT_DRIP_PLUGIN_FILE ) ) );
 		$this->script_suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
 
+		// Handle localisation.
+		$this->load_plugin_textdomain();
+		add_action( 'init', array( $this, 'load_localisation' ), 0 );
+
```

Comment:
Based on https://wordpress.stackexchange.com/a/164138/11614 I learned it's suboptimal to add hooks in the constructor. Your thoughts?

Reply:
Usually avoid it, but usually don't test localizations. I can move it above the deps check in `init()`.

Reply:
7cd45250e0c2f2188dfccf573c0e996d40d76171